### ISSUE TITLE
Doesn't do anything

### ISSUE THREAD

**Issue by fritz-trawa** [SeeLog/python-init-generator#26]
I run it with the context menu on  a folder

![image](https://github.com/user-attachments/assets/f23021ba-f7da-40b6-854c-a5be13a64e61)

It tells me 'Generate 0 __init__.py file(s)`

![image](https://github.com/user-attachments/assets/bc12b7a0-22a3-4d3c-ba7c-e8d33afbb8ab)

**Comment by sisoe24** [comment]
Hey @fritz-trawa! Currently, the extension only generates init files in folders that contain other Python files to avoid creating unnecessary files. But I can see how the right-click option could be useful even without python files.

**Comment by fritz-trawa** [comment]
Thanks Virgil. If it's a new folder it won't have any python files in it yet. You should trust your users know whant they are doing 😉

**Comment by sisoe24** [comment]
> Thanks Virgil. If it's a new folder it won't have any python files in it yet. You should trust your users know whant they are doing 😉

You are right! I'll make an update as soon as I have a chance. Thanks for pointing that out
